src/PostgREST/OpenAPI.hs

```diff
@@ -185,7 +185,9 @@ makePathItem (t, cs, _) = ("/" ++ unpack tn, p $ tableInsertable t)
   where
     tOp = (mempty :: Operation)
       & tags .~ Set.fromList [tn]
-      & description .~ tableDescription t
+      -- Use first line of table description as summary; rest as description (if present)
+      & summary .~ (strip . takeWhile (/='\n') <$> tableDescription t)
```

Comment:
You know, thinking about this again, it might be more efficient to grab both parts of the string in one `breakOn` operation, and then use them later. Like in the `where` clause add a new item:

```haskell
where
  (tSum, tDesc) = fmap fst &&& fmap snd $
                    breakOn "\n" <$> tableDescription t
  -- . . .
  summary .~ mfilter (/="") tSum
  description .~ mfilter (/="") tDesc
```

The weird [&&&](https://hackage.haskell.org/package/base-4.10.0.0/docs/Control-Arrow.html#v:-38--38--38-) thing is letting us move the Maybe inside the tuple.

This may be more efficient than two scans to takeWhile and dropWhile. Do you think the crazy syntax is worth it?